language/move-vm/types/src/data_store.rs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) The Libra Core Contributors
```

Comment:
Hmm so data_store.rs is in move-vm-types not move-vm-state?

Reply:
`move-vm-state` is gone, it's no more....
I am not sure about the `move-vm-types` name but that was always a mix of different things and a bit the standalone API that could be used with and without the runtime.
So not sure about the name but I could move it to `move-vm-runtime`